<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>DAQ</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionVariable_Editor"/>
    <addaction name="actionFile_Viewer"/>
    <addaction name="actionFrame_Viewer"/>
    <addaction name="actionLog_Exporter"/>
    <addaction name="actionCell_Viewer"/>
    <addaction name="actionBootloader"/>
    <addaction name="actionCharge_Viewer"/>
    <addaction name="actionSensor_Viewer"/>
    <addaction name="actionSystem_Monitor"/>
    <addaction name="actionMotor_Viewer"/>
    <addaction name="actionFault_Viewer"/>
    <addaction name="actionCar_Viewer"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Widgets</string>
    </property>
    <addaction name="actionLCD"/>
    <addaction name="actionPlot"/>
    <addaction name="separator"/>
    <addaction name="actionRemoveWidget"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_View"/>
    <addaction name="actionSave_View"/>
   </widget>
   <widget class="QMenu" name="menuEdit_2">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport_log"/>
   </widget>
   <widget class="QMenu" name="menuCAN">
    <property name="title">
     <string>CAN</string>
    </property>
    <addaction name="actionSelect_CAN_Bus"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit_2"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuCAN"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionVariable_Editor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Variable Editor</string>
   </property>
  </action>
  <action name="actionFrame_Viewer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Frame Viewer</string>
   </property>
  </action>
  <action name="actionLCD">
   <property name="text">
    <string>LCD</string>
   </property>
  </action>
  <action name="actionDashboard">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dashboard</string>
   </property>
  </action>
  <action name="actionPlot">
   <property name="text">
    <string>Plot</string>
   </property>
  </action>
  <action name="actionMatPlot">
   <property name="text">
    <string>MatPlot</string>
   </property>
  </action>
  <action name="actionRemoveWidget">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionLog_Exporter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Log Exporter</string>
   </property>
  </action>
  <action name="actionImport_log">
   <property name="text">
    <string>Import Log</string>
   </property>
  </action>
  <action name="actionLoad_View">
   <property name="text">
    <string>Load View</string>
   </property>
  </action>
  <action name="actionSave_View">
   <property name="text">
    <string>Save View</string>
   </property>
  </action>
  <action name="actionCell_Viewer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cell Viewer</string>
   </property>
  </action>
  <action name="actionBootloader">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Bootloader</string>
   </property>
  </action>
  <action name="actionCharge_Viewer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Charge Viewer</string>
   </property>
  </action>
  <action name="actionFile_Viewer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>File Viewer</string>
   </property>
  </action>
  <action name="actionSensor_Viewer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sensor Viewer</string>
   </property>
  </action>
  <action name="actionSystem_Monitor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>System Monitor</string>
   </property>
  </action>
  <action name="actionMotor_Viewer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Motor Viewer</string>
   </property>
  </action>
  <action name="actionFault_Viewer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fault Viewer</string>
   </property>
  </action>
  <action name="actionCar_Viewer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Car Viewer</string>
   </property>
  </action>
  <action name="actionSelect_CAN_Bus">
   <property name="text">
    <string>Select CAN Bus</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
